public/conduct.jade

```diff
@@ -0,0 +1,30 @@
+div.jsla-toplevel-container.jsla-content-page
```

Comment:
also in terms of moving the files nbd but lets try to use `git mv` on the file to rename it. So the original history of the file is not lost. Would be nice to see how the code of conduct changes over the years.